src/util/floatingpoint.cpp

```diff
@@ -14,9 +14,50 @@
  **
  **/
 
-#include "util/floatingpoint.h"
+#include <math.h>
 
 #include "base/cvc4_assert.h"
+#include "util/floatingpoint.h"
+#include "util/integer.h"
+
+#include "symfpu/core/packing.h"
+#include "symfpu/core/compare.h"
+#include "symfpu/core/sign.h"
+#include "symfpu/core/classify.h"
+#include "symfpu/core/add.h"
+#include "symfpu/core/multiply.h"
+#include "symfpu/core/fma.h"
+#include "symfpu/core/divide.h"
+#include "symfpu/core/sqrt.h"
+#include "symfpu/core/convert.h"
+#include "symfpu/core/remainder.h"
+#include "symfpu/core/ite.h"
+#include "symfpu/utils/properties.h"
+#include "symfpu/utils/numberOfRoundingModes.h"
+
+namespace symfpu {
+
+#define CVC4LITITEDFN(T) template <>					\
```

Comment:
Use `CVC4_XXX` for macros, e.g., `CVC4_LIT_ITE_DFN`.

Reply:
Done.